Tasks/PullRequestCodeAnalysis/Tests/L0.ts

```diff
@@ -746,4 +750,53 @@ describe('The PRCA', function () {
             });
         });
     });
+
+    describe('task', () => {
+
+        it('succeeds but skips on a non-PR build', (done:MochaDone) => {
+            this.timeout(1000);
+
+            let tp: string = path.join(__dirname, 'L0skipsNotPr.js');
```

Comment:
This is the new test system in action: "L0skipsNotPr.ts" now runs and sets up the test run, serving much as "response.json" used to in the old test infrastructure with the key change that **getVariable is no longer functional** and manual assignments in process.env is required (examples are in other task tests).